modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -37,6 +27,74 @@ Datacenter: Atlanta
 |openshift-region: us-east-1 (tag), openshift-zone: us-east-1b (tag)
 |This defines a different failure domain within the same region called us-east-1b.
 |===
+
+== Creating vSphere topology during installation or post-installation
+
+=== Procedure
+. In the VMware vCenter vSphere client GUI, define appropriate zone and region catagories and tags.
++
+While vSphere allows you to create categories with any arbitrary name, {product-title} strongly recommends use of `openshift-region` and `openshift-zone` names for defining topology.
++
+For more information about vSphere categories and tags, see the VMware vSphere documentation.
+
+. In {product-title}, create failure domains:
+* xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc#installation-installer-provisioned-vsphere-config-yaml_installing-vsphere-installer-provisioned-customizations[During installation]
+
+* xref:../../post_installation_configuration/post-install-vsphere-zones-regions-configuration.adoc#specifying-regions-zones-infrastructure-vsphere_post-install-vsphere-zones-regions-configuration[Post-installation]
+
+. Create a tag to assign to datastores across failure domains:
++
+When an {product-title} spans more than one failure domain, the datastore might not be shared across those failure domains, which is where topology-aware provisioning of persistent volumes (PVs) is useful. 
++
+.. In vCenter, create a category for tagging the datastores. For example, `openshift-zonal-datastore-cat`. You can use any other category name, provided the category uniquely is used for tagging datastores participating in {product-title} cluster. Also, ensure that `StoragePod`, `Datastore`, and `Folder` are selected as Associable Entities for the created category. 
+.. In vCenter, create a tag that uses the previously created category. This example uses the tag name `openshift-zonal-datastore`.
+.. Assign the previously created tag (in this example `openshift-zonal-datastore`) to each datastore in a failure domain that would be considered for dynamic provisioning.
++
+[NOTE]
+====
+You can use any names you like for categories and tags. The names used in this example are provided as recommendations. Ensure that the tags and categories that you define uniquely identify only datastores that are shared with all hosts in the {product-title} cluster.
+====
```

Comment:
In this section, I guess we suppose user would like to use failure domain defined during installation or post-installation for topology, in that case, I think we don't need to tag the datastores? 
@gnufied Correct me if I'm wrong.

Reply:
Please ignore my previous comments, here we mention assign the tag to datastore which is correct, but I think we could not use any other category name in this section: "Creating vSphere storage topology during installation or post-installation", it should be `openshift-region` and `openshift-zone`, right?

Reply:
I think you are right that this note should be removed from this section (Creating vSphere storage topology during installation or post-installation).

Reply:
We could keep it, depending on how initial failure-domain was configured with different datastores or not. I have pushed a version here -  https://github.com/gnufied/openshift-docs/commit/8d575f78bf662b5901d2b4deb0ac5eef3879afdd

Reply:
Done.